content/src/content/jcr_root/apps/core/wcm/components/navigation/v1/navigation/README.md

```diff
@@ -31,15 +31,15 @@ The Navigation component uses the `com.adobe.cq.wcm.core.components.models.Navig
 The following configuration properties are used:
 
 1. `./navigationRoot` - the root page from which to build the navigation. Can be a blueprint master, language master or regular page.
-2. `./skipNavigationRoot` - if `true`, excludes the navigation root in the resulting tree, including its descendants only.
```

Comment:
I would rather we mark this as deprecated and and replaced by `rootLevel` instead of removing it.